pre_commit_hooks/check_executables_have_shebangs.py

```diff
@@ -60,10 +60,11 @@ def has_shebang(path: str) -> int:
 
 
 def _message(path: str) -> None:
+    is_windows = sys.platform == 'win32'
     print(
         f'{path}: marked executable but has no (or invalid) shebang!\n'
         f"  If it isn't supposed to be executable, try: "
-        f'`chmod -x {shlex.quote(path)}`\n'
+        f'`{"git add --chmod=-x" if is_windows else "chmod -x"} {shlex.quote(path)}`\n'
```

Comment:
ah I mean, without having any branching logic just write:

"chmod -x ..."
" if on windows you may also need to ..."